### ISSUE TITLE
General - IOS and Android

### ISSUE THREAD

**Issue by Collabophotos** [nautilusdigital/collabo-issues#17]
When creating an event, can we have a pop up or something that tells the user you can't have an end date that is before the start date?  It won't let you create an event with that which is good, but it doesn't tell the user why they couldn't create the event.  It may just seem like it's not working if the user doesn't realize the end date is set before the start date.

**Comment by mhousser** [comment]
Agree. The server validates the dates, but the apps should do this as well to prevent wasted time and confusion.

Some warning message like `The end date can't be before the start date` if both are entered, etc. @vivanov1410 or @aethusn .

**Comment by Collabophotos** [comment]
1. When inviting people to event, it allows me to invite someone who is already accepted and is already a member.  It should prevent you from inviting someone already part of group
2.When uploading additional pictures, it auto selects ones that user has already uploaded. It should recognized duplicate photos and indicate somewhere already uploaded when showing gallery otherwise user has to uncheck previously uploaded photographs.

**Comment by Collabophotos** [comment]
This is an enhancement - ability to create distribution lists so that every time you create an event you do not have to manually invited each person again.  People typically go to events with same handful of people.

**Comment by vivanov1410** [comment]
👍
